src/ocl/convolutionocl.cpp

```diff
@@ -768,7 +768,6 @@ void ConvolutionDescriptor::FindConvFwdAlgorithm(Handle& handle,
 
     std::vector<PerfField> perf_db;
 
-    const miopen::FindMode fm(ctx);
```

Comment:
This shouldn't be removed. The `findMode` should be constructed locally from the find mode in the convolution descriptor so it can be overridden when the user pass `true` for `exhaustiveSearch`.

Reply:
> so it can be overridden when the user pass `true` for `exhaustiveSearch`.

Not feasible without API change. See https://github.com/ROCmSoftwarePlatform/MIOpen/pull/489#issuecomment-706332733 and https://github.com/ROCmSoftwarePlatform/MIOpen/pull/489#issuecomment-706382083.